mkt/abuse/migrations/0003_abusereport_website.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('abuse', '0002_auto_20150722_1517'),
+        ('websites', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='abusereport',
+            name='website',
+            field=models.ForeignKey(related_name='abuse_reports', to='websites.Website', null=True),
+            preserve_default=True,
+        ),
+    ]
```

Comment:
I'll give it a try. It generated all 3 at once, so I don't know if there's a load-order problem it's trying to avoid.